PurchasesCoreSwift/LocalReceiptParsing/DataConverters/UInt8+Extensions.swift

```diff
@@ -8,28 +8,52 @@
 
 import Foundation
 
+// swiftlint:disable identifier_name
+enum BitShiftError: Error {
```

Comment:
which one is swiftlint complaining about here?

Reply:
`to` being shorter than 3 characters. I was thinking about setting the min length to 3 chars, but decided that we so infrequently will run into a valid use case for `to` that I should just disable it for this one file, for now.

Reply:
👍